app/src/main/java/ch/epfl/sdp/musiconnect/pages/MyProfilePage.java

```diff
@@ -139,7 +128,6 @@ public void onClick(View view) {
         super.displayNotFinishedFunctionalityMessage();
     }
 
-
```

Comment:
I'm not sure we need this onClick anymore at line 138.

Reply:
I think @antoniopisanell took it out in his pr

Reply:
Actually we need to override it because of this `implements View.OnClickListener`

Reply:
Ah they were used to play videos. But now they're not used since the video plays automatically